CRM/Member/Form/MembershipRenewal.php

```diff
@@ -741,14 +740,9 @@ public function processMembership($memParams, $changeToday, $numRenewTerms, $pen
       // CRM-15475
       array_search('Cancelled', CRM_Member_PseudoConstant::membershipStatus(NULL, " name = 'Cancelled' ", 'name', FALSE, TRUE)),
     ])) {
-      $memParams = array_merge($memParams, [
-        'status_id' => $currentMembership['status_id'],
-        'start_date' => $currentMembership['start_date'],
-        'end_date' => $currentMembership['end_date'],
-      ]);
       return CRM_Member_BAO_Membership::create($memParams);
     }
-
+    $memParams['join_date'] = date('Ymd', strtotime($currentMembership['join_date']));
```

Comment:
@eileenmcnaughton any reason why use Ymd rather than `Y-m-d`  here?

Reply:
@seamuslee001 I don't really see either being better than the other

Reply:
Just thought we were standardising on using `Y-m-d` as the format when passing to the BAO

Reply:
@seamuslee001 it looks to me like apiv3 sets to Ymd https://github.com/civicrm/civicrm-core/blob/d33b612186671c84ff157675a176a55b754aac09/api/v3/utils.php#L1703